apps/skde/pages/pasientstrommer/index.tsx

```diff
@@ -3,13 +3,13 @@ import { Footer } from "../../src/components/Footer";
 import { ThemeProvider } from "@mui/material";
 import { skdeTheme } from "qmongjs";
 import { PageWrapper } from "../../src/components/StyledComponents/PageWrapper";
-import { Header } from "../../src/components/Header";
+import { SkdeHeader } from "../../src/components/Header";
```

Comment:
Ville beholdt navnet `HeaderTop` her for å skille den fra `HeaderMiddle` og siden dette i grunn er en FNSP-header